googleapiclient/errors.py

```diff
@@ -107,6 +107,10 @@ class InvalidNotificationError(Error):
   """The channel Notification is invalid."""
   pass
 
+class InvalidMessageError(Error):
```

Comment:
The name InvalidMessage is more generic than what the docstring says. Maybe you can change the docstring a bit to match the name?